ajax/libs/venn.js/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "venn.js",
+  "filename": "venn.min.js",
+  "description": "A javascript library for laying out area proportional venn and euler diagrams",
+  "homepage": "https://benfred.github.io/venn.js/",
+  "author": "Ben Frederickson (ben@benfrederickson.com)",
+  "keywords": [
+    "venn-diagram",
+    "javascript",
+    "d3",
+    "euler-diagram",
+    "venn",
+    "euler"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/benfred/venn.js.git"
+  },
+  "license": "MIT",
+  "npmName": "venn.js",
+  "npmFileMap": [
+    {
+      "basePath": "build",
```

Comment:
The files in this directory will clash with those at the root:

```
├── build
│   ├── venn.js
│   └── venn.min.js
└── venn.js
```

Reply:
I think the best option may be to just serve `build` only.

Reply:
addressed in 009cf008e9cd8af3f75ee63fd687bd4e3ff6a8a8